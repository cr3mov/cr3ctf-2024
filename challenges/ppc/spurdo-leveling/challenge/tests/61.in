136127306854637528 964864217034943029
81 799364243434
94 996559860944
