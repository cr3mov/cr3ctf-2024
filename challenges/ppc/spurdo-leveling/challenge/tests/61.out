34962499 582708 364194
